### PR TITLE
Product copy text changes v1

### PR DESCRIPTION
**Pull request by vincentbpro**
### What does it do?

Text changes for the most part

Other changes:
- Adds back link on the CM page
- Adds dynamic "get help" link under the version in settings/overview
- Adds call to action when the content list is empty in the CM
- Makes the "test email" button disabled until a valid email address is provided in settings/email/configuration

### CODE REVIEW

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/index.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ const ListPage = () => {
         subtitle={formatMessage(
           {
             id: 'Settings.permissions.users.listview.header.subtitle',
-            defaultMessage: '{number, plural, =0 {# users} one {# user} other {# users}} found',
+            defaultMessage: 'All the users who have access to the Strapi admin panel',
           },
           { number: total }
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/index.js:125]: I think you don't need the number variable in here anymore?

File: packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js [hunk 2]
```diff
@@ -61,8 +60,7 @@ const LeftMenu = () => {
       id: 'singleTypes',
       title: {
         id: getTrad('components.LeftMenu.single-types'),
-        defaultMessage:
-          '{number, plural, =0 {Single Types} one {Single Type } other {Single Types}}',
+        defaultMessage: 'Single Types',
         values: { number: intlSingleTypeLinks.length },
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js:64]: Same here I think you don't use the number variable anymore!

Comment (vincentbpro) [reply packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js:64]: Very true, thanks! Will check and clean all of them :+1:

**Review by HichamELBSI — DISMISSED** [review]
LGTM

**Review by HichamELBSI — APPROVED** [review]
👍🏽  Good job

**Comment by HichamELBSI** [comment]
@vincentbpro Can you update the branch please 😉

**Comment by derrickmehaffy** [comment]
This pull request has been mentioned on **Strapi Community Forum**. There might be relevant details there:

https://forum.strapi.io/t/implement-back-button-in-local-plugin/13978/2

**Pull request merged**